openfe/setup/network.py

```diff
@@ -1,13 +1,64 @@
 import math
+import networkx as nx
+from rdkit import Chem
+from typing import TypeVar
+
+
+NetworkXGraph = TypeVar("NetworkXGraph")
+
+
+def _hashmol(mol):
```

Comment:
move to public method in utils? this comes back to my sense that we need an actual hashable molecule class, because it will be reused in so many places, and users will want to use molecules as keys in dicts.